dist/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Meals app</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+<script defer src="index.bundle.js"></script></head>
+
+<body>
+  <nav class="container border border-dark border-3 mt-4 navbar navbar-expand navbar-light bg-white menu">
+    <div class="container-fluid col-12 col-md-12">
+      <a href="#home"><img src="#" alt="" id="logo" class="navbar-brand col-4"></a>
+      <ul class="navbar-nav ms-auto d-flex align-items-baseline col-8 justify-content-end">
+        <li class="nav-item mx-lg-2 mx-xl-3 mx-xxl-4">
+          <a class="nav-link" href="#home" id="list-identifier">Home</a>
+        </li>
+        <li class="nav-item mx-lg-2 mx-xl-3 mx-xxl-4">
+          <a class="nav-link" href="#ingredients" id="add-identifier">Ingredients</a>
+        </li>
+        <li class="nav-item ml-lg-2 ml-xl-3 ml-xxl-4">
+          <a class="nav-link" href="#about" id="contact-identifier">About</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
```

Comment:
![image](https://user-images.githubusercontent.com/68148157/143071923-786e35a4-67f9-44c1-9b17-5cc59f1dd2da.png)

The font looks smaller. Increasing the font will increase the visibility

Reply:
Font size increased 👍